CONTRIBUTING.md

```diff
@@ -0,0 +1,27 @@
+Contributing to Ansible-Satellite6
+==================================
+
+To contribute to ansible-satellite6, please use pull requests on a branch of your own fork.
+
+After creating your fork on github, you can do:
+```
+git clone git@github.com:yourname/ansible-satellite6
+cd ansible-satellite6
+git checkout -b your-branch-name
```

Comment:
Should we have them add the remote so they can rebase easily and pull in new changes?

Reply:
That suggestion is based on this link, https://help.github.com/articles/fork-a-repo/ :)